src/components/EditCoursePage/index.jsx

```diff
@@ -78,8 +78,7 @@ class EditCoursePage extends React.Component {
     /* eslint-enable no-param-reassign */
   }
 
-  prepareCourse(courseData) {
-    /* eslint-disable no-param-reassign */
+  prepareCourse(courseEditData) {
```

Comment:
Nit: Keep the input parameter as courseData since that is what we are passing in.

Just felt a little weird to me when the input parameter of the function is courseEditData and the value to assign the function to is courseEditData (line 139)